### PR TITLE
Add temporary drawer

### PR DESCRIPTION
**Pull request by ceciliamvrie**
Modified: 
* Drawer

I re-did the original PR - this one doesn't modify the persistent drawer,
but adds support and styles for a temporary drawer. Thanks @martahj for showing
the correct implementation of how different drawers should be like!

link => https://docs-cdikxdudmw.now.sh!

### CODE REVIEW

File: pages/appbar.js [hunk 1]
```diff
@@ -52,6 +66,13 @@ export default class AppbarPage extends Component {
             >
               Launch Appbar Drawer
```
Comment (ashwoodall) [inline pages/appbar.js]: If we're adding different types of drawers to the appbar page, should we make a clear difference between the button labels?

Comment (ceciliamvrie) [reply pages/appbar.js]: Yeah! For the permanent drawer button I can put 'Launch Permanent Drawer'

File: pages/appbar.js [hunk 2]
```diff
@@ -38,12 +44,20 @@ export default class AppbarPage extends Component {
...
               <Appbar title='I am in this drawer!' />
-              <p>{'hello!'}</p>
+              <p>I am a <b>persistent</b> drawer!</p>
+            </Drawer>
+            <Drawer
+              temporary
+              attachment='right'
+              open={this.state.temporaryRight}
+              handleRequestClose={this.toggleTemporaryRight}>
+              <Appbar title='I am a temporary appbar!' />
```
Comment (ashwoodall) [inline pages/appbar.js]: one more NIT: <em>technically</em> the appbar isn't temporary, but the drawer is

Comment (ceciliamvrie) [reply pages/appbar.js]: Hmm, there seems to be different app bars depending on the type of drawer that's with it - whether it's permanent or it's temporary 
https://material.io/guidelines/patterns/navigation-drawer.html#navigation-drawer-content

Comment (ashwoodall) [reply pages/appbar.js]: Is that the subheader component though?

Comment (ceciliamvrie) [reply pages/appbar.js]: I see what you mean. I'll change it, thanks!

**Pull request merged**